stl/inc/yvals_core.h

```diff
@@ -248,6 +248,7 @@
 // P2259R1 Repairing Input Range Adaptors And counted_iterator
 //     (partially implemented)
 // P2325R3 Views Should Not Be Required To Be Default Constructible
+// P2367R0 Remove Misuses Of list-initialization From Clause 24 Ranges

```

Comment:
I don't understand the capitalization scheme here.

Reply:
I presume Start Case except for standard wording like `list-initialization` and `counted_iterator` (seen above)?

Reply:
I honestly can't remember why I thought that list-initialization was special enough to be immune to Title Case Everything Except `identifiers`. It does look weird now that you point it out.

Reply:
> I honestly can't remember why I thought that list-initialization was special enough to be immune to Title Case Everything Except `identifiers`. It does look weird now that you point it out. I'd be happy with updating the linked issue and the comment here to say List-Initialization.

You didn't: I vetoed your uppercasing of the word of power "list-initialization" - it's important that words of power appear exactly as they do in the working draft. That said, I find no support for my belief that capitalization is important in the working draft. In fact, the defining occurrence of list-initialization in [[dcl.init.list]/1](https://eel.is/c++draft/dcl.init.list#1) is capitalized. I'll revert.